langchain4j-embeddings/src/main/java/dev/langchain4j/model/embedding/OnnxBertBiEncoder.java

```diff
@@ -1,5 +1,7 @@
 package dev.langchain4j.model.embedding;
 
+import ai.djl.huggingface.tokenizers.Encoding;
+import ai.djl.huggingface.tokenizers.HuggingFaceTokenizer;
```

Comment:
Is this right? A `HuggingFaceTokenizer` class is being created in this PR, but then never used (beyond tests). Is this import supposed to be here instead of `dev.langchain4j.model.embedding.HuggingFaceTokenizer`?

Asking because I'm getting:

```
java.lang.AssertionError: No tokenizers version found in property file.
	at ai.djl.util.Platform.detectPlatform(Platform.java:94)
	at ai.djl.huggingface.tokenizers.jni.LibUtils.copyJniLibraryFromClasspath(LibUtils.java:82)
	at ai.djl.huggingface.tokenizers.jni.LibUtils.loadLibrary(LibUtils.java:66)
	at ai.djl.huggingface.tokenizers.jni.LibUtils.<clinit>(LibUtils.java:41)
	at ai.djl.huggingface.tokenizers.HuggingFaceTokenizer.newInstance(HuggingFaceTokenizer.java:173)
	at dev.langchain4j.model.embedding.OnnxBertBiEncoder.<init>(OnnxBertBiEncoder.java:39)
	at dev.langchain4j.model.embedding.AbstractInProcessEmbeddingModel.loadFromJar(AbstractInProcessEmbeddingModel.java:20)
	at dev.langchain4j.model.embedding.AllMiniLmL6V2EmbeddingModel.<clinit>(AllMiniLmL6V2EmbeddingModel.java:18)
```

Reply:
@amuniz this seems to be correct. `dev.langchain4j.model.embedding.HuggingFaceTokenizer` is supposed to be used by users, for example, when using `TokenWindowChatMemory`.

How do I reproduce this exception?
Thanks!